Kickstarter-iOS/Features/PledgeView/Controllers/NoShippingPledgeViewController.swift

```diff
@@ -108,6 +108,11 @@ final class NoShippingPledgeViewController: UIViewController,
       |> \.translatesAutoresizingMaskIntoConstraints .~ false
   }()
 
+  private lazy var estimatedShippingStackView: UIStackView = {
+    UIStackView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
```

Comment:
@ifosli For the correct margins to be applied, the estimated shipping view needs to be added to a new stack view as a subview. This is true for UIKit views as well. I spent time rebuilding the view with UIKit to verify this.